silhouette/app/com/mohiva/play/silhouette/impl/providers/OAuth2Provider.scala

```diff
@@ -142,12 +145,16 @@ trait OAuth2Provider extends SocialProvider with OAuth2Constants with Logger {
    * @return The info containing the access token.
    */
   protected def getAccessToken(code: String)(implicit request: RequestHeader): Future[OAuth2Info] = {
-    httpLayer.url(settings.accessTokenURL).withHeaders(headers: _*).post(Map(
+    val params = Map(
```

Comment:
The script does not format such lines. Do you have formatted the code? Or does your IDE has formatted the code automatically?